src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddleware.cs

```diff
@@ -32,6 +34,7 @@ public class DeveloperExceptionPageMiddleware
         private readonly DiagnosticSource _diagnosticSource;
         private readonly ExceptionDetailsProvider _exceptionDetailsProvider;
         private readonly Func<ErrorContext, Task> _exceptionHandler;
+        private static readonly MediaTypeHeaderValue _textPlainMediaType = new MediaTypeHeaderValue("text/html");
```

Comment:
Shouldn't the value be `text/plain`?

Reply:
Then the variable name is a bit missleading.

Reply:
oops!

Reply:
@davidfowl seems that the lines below seem to be wanting `text/plain` and not `text/html`:

https://github.com/aspnet/AspNetCore/blob/9bcb63c65495e6ad074258fd959183d49d5f703a/src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddleware.cs#L138-L140

Reply:
?

Reply:
Never-mind me -- I mixed up the initial PR content with the latest code, missing the changes you committed in the meantime. All 👍 now...